src/credentials/providers/credentialsProviderManager.ts

```diff
@@ -0,0 +1,61 @@
+/*!
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { CredentialsProvider } from './credentialsProvider'
+import { CredentialsProviderFactory } from './credentialsProviderFactory'
+import { makeCredentialsProviderIdComponents } from './credentialsProviderId'
+
+let credentialsProviderManagerInstance: CredentialsProviderManager | undefined
+
+export function getCredentialsProviderManagerInstance(): CredentialsProviderManager {
```

Comment:
Can we put a `getInstance` on `CredentialsProviderManager` instead?

Reply:
Sure